core/async/src/test_loop/event_handler.rs

```diff
@@ -1,40 +1,125 @@
-use std::time::Duration;
+use near_primitives::time::{self, Clock, Duration};
```

Comment:
With short meaningful module names, I recommend importing just the module name and then use qualified names for their content. At least, that's the convention that I use.

Reply:
I struggled quite a bit here in picking which style:
 - If we import near_primitives::time and then use time::everything everywhere, like what you do in near_network, the spelling is quite verbose, and migrating to use this crate is a lot more work than changing the import. Also, "time::Duration" would work even without the module import, but would refer to the "::time::Duration", which can lead to confusion or actually using the wrong type (well, right now they are the same, so we wouldn't even catch that).
 - The other approach is just importing near_primitives::time::{Duration, Instant, etc.}, and standardize the whole codebase so that Duration and Instant only refer to near_primitives::time::*. It seems then there's no real need to spell out time::* since there's only one such Duration and one such Instant. I'm thinking then we can ban the use of any other Duration or Instant by using a clippy disallow_types rule.

What do you think?